internal/pkg/cli/storage_init.go

```diff
@@ -46,15 +47,88 @@ We'll deploy the Cloudformation for the storage when you run 'svc deploy'.`
 
 var fmtStorageInitCreateConfirm = "Okay, we'll create %s %s named %s linked to your %s service."
 
+var fmtStorageInitDynamoInfoText = `We're going to ask you some questions about your database schema and datatypes.
+You can change any of this information at the end by editing the CloudFormation
+generated at ./ecs-project/%s/addons/%s-%s.yml.`
+
+var (
+	fmtStorageInitDDBKeyPrompt  = "What would you like to name the %s of this %s?"
+	storageInitDDBKeyTypePrompt = "What datatype is this key?"
+	storageInitDDBLSIPrompt     = "Would you like to add a local secondary index to this table?"
+	storageInitDDBLSINamePrompt = "What would you like to name this " + color.Emphasize("LSI") + "?"
+)
+
```

Comment:
Is there a reason why we split the groups like this? if so can we add comments to the top of each group.

One thing that'd be helpful for me is seeing all the prompts regarding a particular field together like:
```go
var (
  fmtStorageInitDDBKeyPrompt =
  storageInitDDBKeyTypePrompt =
  storageInitDDBKeyTypeHelp =
  
  storageInitDDBLSIPrompt = 
  storageInitDDBLSIHelp = 
  ...
)
```